pom.xml

```diff
@@ -44,6 +44,7 @@
         <unirest.version>3.10.00</unirest.version>
         <bucket4j-starter.version>0.2.0</bucket4j-starter.version>
         <caffeine.version>2.8.5</caffeine.version>
+        <sentry.version>1.6.3</sentry.version>
```

Comment:
Add a comment here not to update this, unless Sentry instance was updated. I will forget about not upgrading this otherwise

Reply:
but i will not! and you need approve :P

Reply:
Okay, in such cases I will blame you! :D